dev/com.ibm.ws.logstash.collector_fat/fat/src/com/ibm/ws/logstash/collector/tests/CustomizedTagTest.java

```diff
@@ -64,39 +54,27 @@
 
     private final String testName = "";
     private static Class<?> c = CustomizedTagTest.class;
-    public static String pathToAutoFVTTestFiles = "lib/LibertyFATTestFiles/";
     private static String os = "";
-    private static String JVMSecurity = System.getProperty("Djava.security.properties");
 
-    protected static boolean runTest;
+    protected static boolean runTest = true;
```

Comment:
Can we just remove all the runTest variables since it will always be true?

Reply:
I kept them just in case we need to skip some platforms.  We can remove them after a few weeks of successful runs in SOE.